File: src/main/java/com/cse3063f25grp1/model/Query.java
package com.cse3063f25grp1.model;

public class Query {
    private String text;

    public Query() {
    }

    public Query(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }

    @Override
    public String toString() {
        return "Query{" +
                "text='" + text + '\'' +
                '}';
    }
}

File: iteration2/src/model/query.py
class Query:
    def __init__(self, text: str = None):
        self.__text = text

    # Getter
    def get_text(self) -> str:
        return self.__text

    # String representation
    def __str__(self) -> str:
        return f"Query{{text='{self.__text}'}}"
    
